release notes/v0.40.0.md

```diff
@@ -0,0 +1,270 @@
+
+# Maintenance and internal improvements
+
+- [#2590](https://github.com/grafana/k6/pull/2590) Update direct dependencies without any interesting changes apart goja.
+- [#2591](https://github.com/grafana/k6/pull/2591) Changes to the CI process to always build rpm/deb and windows packages and use nfpm to do it. 
+- [#2593](https://github.com/grafana/k6/pull/2593) Internal cleanup after finally removing `common.Bind`.
+- [#2597](https://github.com/grafana/k6/pull/2597) Fix go benchmarks we have broken over time.
+- [#2599](https://github.com/grafana/k6/pull/2599) Reformat `//nolint` comments as part of getting ready for go 1.19.
+- A bunch of fixes for tests [#2589](https://github.com/grafana/k6/pull/2589), [#2620](https://github.com/grafana/k6/pull/2620), [#2625](https://github.com/grafana/k6/pull/2625), [#2643](https://github.com/grafana/k6/pull/2543), [#2647](https://github.com/grafana/k6/pull/2647), [#2648](https://github.com/grafana/k6/pull/2648),
+- [#2607](https://github.com/grafana/k6/pull/2607) Fix build badge in the README. Thanks @AetherUnbound!
+- [#2614](https://github.com/grafana/k6/pull/2614) Fix advice for rpm install on Amazone Linux.
+- [#2615](https://github.com/grafana/k6/pull/2615) Improve documentation of RegisterCallback due to feedback on how hard it was to understand how to use it properly.
+- [#2627](https://github.com/grafana/k6/pull/2627) Create distinct test state objects for the pre-init and run phases.
+- [#2635](https://github.com/grafana/k6/pull/2635) Drop License header in each file.
+- [#2636](https://github.com/grafana/k6/pull/2636) Add [SECURITY.md](https://github.com/grafana/k6/security/policy) with instructions how to report security issues responsibly.
+- [#2641](https://github.com/grafana/k6/pull/2641) Fix spelling of `lose`. Thanks @spazm!
+- Update to golangci-lint v1.47.2 and enable a bunch more linters. [#2609](https://github.com/grafana/k6/pull/2609), [#2611](https://github.com/grafana/k6/pull/2611). Also, drop obsolete configurations [#2619](https://github.com/grafana/k6/pull/2619).
+
+# Roadmap and future plans
+
+This section will talk about our plans for future versions. You will notice that two of the big ticket items are here again. This is mostly due to the sheer size of the work required on both and some unforeseen needed changes - most of them actually making this better in the long run. It also so happens that it is vacation season so the k6 team rightfully is taking some personal time away.
+
+## Native support for ECMAScript modules
+
+The native ESM support is coming. There is both under development [PR to k6](https://github.com/grafana/k6/pull/2563) and a branch that will become a PR to goja to add the support there. The k6 team is hopeful that this will land in the next version v0.41.0!
```

Comment:
```suggestion
The native ESM support is coming. A [PR to k6](https://github.com/grafana/k6/pull/2563) is under development, and a branch that will become a PR to goja to add the support there. The k6 team is hopeful that this will land in the next version, v0.41.0!
```